src/mgmt/srv.pb-c.c

```diff
@@ -0,0 +1,1434 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: srv.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "srv.pb-c.h"
+void   proto__empty_params__init
+                     (Proto__EmptyParams         *message)
+{
+  static const Proto__EmptyParams init_value = PROTO__EMPTY_PARAMS__INIT;
+  *message = init_value;
+}
+size_t proto__empty_params__get_packed_size
```

Comment:
Im not sure why these are prefaced with proto while the security ones aren't when they both have proto as the package. I think it has to do with the path used to generate the C source fils. I'll have to look into it.